TSPL.docc/GuidedTour/AboutSwift.md

```diff
@@ -2,23 +2,19 @@
 
 Understand the high-level goals of the language.
 
-Swift is a fantastic way to write software,
-whether it’s for phones, desktops, servers,
+Swift is a fantastic way to write software
+for phones, tablets, desktops, servers,
 or anything else that runs code.
-It's a safe, fast, and interactive programming language
+It's a safe and fast programming language
 that combines the best in modern language thinking
-with wisdom from the wider Apple engineering culture
-and the diverse contributions from its open-source community.
-The compiler is optimized for performance
-and the language is optimized for development,
-without compromising on either.
+with wisdom from a diverse open source community.
 
 Swift is friendly to new programmers.
```

Comment:
I think this paragraph works as-is if you read these first two paragraphs as "the elevator pitch" for Swift, which I believe was the intent e.g. Swift is friendly because it's an industrial-quality language that is still expressive and enjoyable; it manages to be performant and optimized for the developer without compromise.

Curious what @amartini51 thinks.

Reply:
This paragraph might hold together better if we expanded the first sentence a bit.  Part of the framing is that Swift scales from simple to complex use cases.  Some languages like BASIC were designed only with beginners in mind, and experienced programers tend to move on to other languages — Swift was designed to work for both.

> Swift is friendly to new programmers,
> and powerful enough for experienced programmers.

Or

> Swift is friendly to new programmers,
> without sacrificing the power and flexibility that experienced programmers need.

Reply:
Changed in ae732c2e.